### PR TITLE
Feat/479653  failed status tasklist

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

File: app/lib/manage-cdo-helpers.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const { statuses } = require('../constants/cdo/status')
```
Comment (jbarnsley10) [inline app/lib/manage-cdo-helpers.js:1]: These methods would have sat nicely in route-helpers but you can keep them here

Comment (whitewaterdesign) [reply app/lib/manage-cdo-helpers.js:1]: good point, happy to move later

File: app/frontend/css/application.scss [hunk 2]
```diff
@@ -88,14 +88,25 @@ $govuk-global-styles: true; // https://frontend.design-system.service.gov.uk/sas
     }
   }
 
-//defra header
 .defra {
 
+  &-responsive {
```
Comment (jbarnsley10) [inline app/frontend/css/application.scss:93]: I don;t think this is working exactly how Nick wants it but we can iterate in a later PR.

**Review by jbarnsley10 — APPROVED** [review]
Looks good. As said below, think we'll need to revisit the CSS but let's get this one out.

**Pull request merged**